openmetadata-ui/src/main/resources/ui/src/components/Dashboard/DashboardDetails/DashboardDetails.component.tsx

```diff
@@ -302,6 +315,15 @@ const DashboardDetails = ({
               className="tabs-new"
               data-testid="tabs"
               items={tabs}
+              tabBarExtraContent={
+                isExpandViewSupported && (
+                  <AlignRightIconButton
+                    className={isTabExpanded ? 'rotate-180' : ''}
+                    size="small"
+                    onClick={toggleTabExpanded}
+                  />
+                )
+              }
```

Comment:
it will send either TabBarExtraContent or false type of data, is that expected? i think we should explicitly send undefined if isExpandViewSupported is false

Reply:
It accepts a React node and React node can be boolean so all good here. It also working as expected
<img width="690" alt="Screenshot 2025-04-02 at 11 58 12 AM" src="https://github.com/user-attachments/assets/dc5072b3-1153-4f6e-9790-79d10d1b752b" />